### ISSUE TITLE
Unable to run just elkstack::forwarder on node, logstash user missing

### ISSUE THREAD

**Issue by markcallen** [rackspace-cookbooks/elkstack#143]
Just trying to install forwarder on a node using the following kitchen suite:

````
  - name: forwarder
    data_bags_path: "default/data_bags"
    encrypted_data_bag_secret_key_path: "default/encrypted_data_bag_secret"
    run_list:
      - recipe[elkstack::forwarder]
    attributes:
      elasticsearch:
        discovery:
          zen:
            ping:
              unicast:
                hosts: ["logstash.getuhost.org"]
````
but I get:
````
       [2015-05-07T21:20:09+00:00] ERROR: Running exception handlers
       Running handlers complete
       [2015-05-07T21:20:09+00:00] ERROR: Exception handlers complete
       [2015-05-07T21:20:09+00:00] FATAL: Stacktrace dumped to /tmp/kitchen/cache/chef-stacktrace.out
       Chef Client failed. 17 resources updated in 710.256224695 seconds
       [2015-05-07T21:20:09+00:00] ERROR: file[/opt/logstash/lumberjack.key] (elkstack::_lumberjack_secrets line 56) had an error: Chef::Exceptions::UserIDNotFound: cannot determine user id for 'logstash', does the user exist on this system?
       [2015-05-07T21:20:10+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
````
Any ideas on which recipe should create this user, or is there one in logstash I could use before the forwarder?

**Comment by martinb3** [comment]
Thank you for reporting this. We're testing with systems that contain a uid/gid for a logstash user already. I hope to fix this with #145.

**Comment by martinb3** [comment]
Hello! I've released 5.0.1 which should default the lumberjack-related keypair to a different location. Thank you!
